fix: mention ACTUAL_DATA_DIR is now dataDar in config (#758)
actualbudget/docs: docs/actual-server-repo-move.md

  (If you are on Windows, you'll need to install [Git Bash](https://git-scm.com/download).)

  1. Open Bash, then Clone the [Actual repository](https://github.com/actualbudget/actual). You can use the following command:

  ```
  git clone https://github.com/actualbudget/actual.git
  ```

  2. Navigate to the Actual project root directory:

  ```
  cd actual
  ```

  3. Install the required dependencies for the server:

  ```
  yarn install
  ```

  4. Build the server:

  ```
  yarn build:server
  ```

>   5. If you have a [config.json](docs/config/) file you will need to copy it into the following directory:

Reviewer: :::suggestion
  5. If you have a [config.json](/docs/config/) file you will need to copy it into the following directory:
:::

  ```
  packages/sync-server
  ```

>   There might be adjustments to make (e.g., `ACTUAL_DATA_DIR` becomes `dataDir`), please refer to [troubleshooting the server](./troubleshooting/server.md) for some help

Reviewer: This is a passive sentence - please make it active. Proposal:

```suggestion
You may need to make some adjustments (e.g., `ACTUAL_DATA_DIR` becomes `dataDir`), please refer to [troubleshooting the server](./troubleshooting/server.md) for some help
```

> You may need to make some adjustments (e.g., `ACTUAL_DATA_DIR` becomes `dataDir`), please refer to [troubleshooting the server](./troubleshooting/server.md) for some help

Reviewer: We use absolute paths, eg. docs/troubleshooting/server.md
